Microsoft.HealthVault/Connection/IConnectionInternal.cs

```diff
@@ -10,6 +10,8 @@
 using System.Collections.Generic;
 using Microsoft.HealthVault.PlatformInformation;
 using Microsoft.HealthVault.Transport;
```

Comment:
That is what I mean, you have System, then Microsoft, then System right?

---
In reply to: [115857722](https://github.com/Microsoft/healthvault-dotnetstandard-sdk/pull/27#discussion_r115857722) [](ancestors = 115857722,115855562)